<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>TYPE</string>
      </property>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>0</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>IP ADDRESS</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>0</y>
         <width>151</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>EMAIL ADDRESS</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>0</y>
         <width>151</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>PHONE NUMBER</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>All files selected :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="FileList"/>
    </item>
    <item>
     <widget class="QPushButton" name="AddFile">
      <property name="text">
       <string>Find a file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Send file</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>AddFile</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>AddFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>9</x>
     <y>546</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sendFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>538</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>FileList</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>522</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>AddFile()</slot>
  <slot>sendFile()</slot>
  <slot>box1()</slot>
  <slot>box2()</slot>
  <slot>box3()</slot>
 </slots>
</ui>
